stories/__tests__/SQFormDialog.stories.test.js

```diff
@@ -96,6 +96,31 @@ describe('Tests for Default', () => {
   });
 });
 
+it('should not call onClose on `escape` keydown because cancel is not available', async () => {
```

Comment:
This test looks solid to me. To fully test this code enhancement I think we need the test where we set `showSecondaryButton={false}` and then we expect the Close button to NOT be in the document or visible.

Reply:
In this test we set `showSecondaryButton={false}` and then query the screen for the button and assert it doesn't exist.  Is that what you are outlining?

Reply:
> In this test we set `showSecondaryButton={false}` and then query the screen for the button and assert it doesn't exist. Is that what you are outlining?

I think this is the same question we discussed in Slack. (We wan't to test whether or not the button appears on the screen as its own test)